forgerock-auth/src/test/java/org/forgerock/android/auth/FRUserMockTest.java

```diff
@@ -207,6 +207,36 @@ public void testAccessToken() throws Exception {
         assertNotNull(accessToken.getValue());
     }
 
+    @Test
+    public void testRevokeAccessToken() throws Exception {
+        frUserHappyPath();
+        
+        //Revoke the token and check the listener onSuccess and onFailure methods
+        FRUser.getCurrentUser().revokeAccessToken(new FRListener<Void>() {
```

Comment:
Since this is an async call, this code will always complete without fail, u need to use CountdownLatch to wait for the result.

Reply:
Had my suspicion that this might be the case. Let's have a quick call when you are online to go through an example of that as I am not very familiar with unit tests in Android. 
Cheers

Reply:
I think I found the way, please let me know if this correct or not :)